pkgs/tools/games/minecraft/mcaselector/default.nix

```diff
@@ -17,7 +18,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   dontUnpack = true;
   dontBuild = true;
 
-  nativeBuildInputs = [ jre makeWrapper ];
+  nativeBuildInputs = [ jre makeWrapper wrapGAppsHook ];
```

Comment:
Sorry for the late reply. Please add `dontWrapGApps = true` and append `${gappsWrapperArgs[@]}` to `makeWrapper` to avoid double wrapping.